frontend/src/tabs/authorship.pug

```diff
@@ -98,17 +105,30 @@
                 font-awesome-icon.button(icon="history")
                 span.tooltip-text Click to view the history view of file
             a(
-              v-bind:href="getFileLink(file, 'blame')", target="_blank"
+              v-if='!file.isBinary',
+              v-bind:href="getFileLink(file, 'blame')", target="_blank",
+              title="click to view the blame view of file"
             )
               .tooltip
                 font-awesome-icon.button(icon="user-edit")
                 span.tooltip-text Click to view the blame view of file
           span.fileTypeLabel(
+            v-if='!file.isBinary',
             v-bind:style="{\
               'background-color': fileTypeColors[file.fileType],\
               'color': this.getFontColor(fileTypeColors[file.fileType])\
               }"
-          ) {{ file.fileType }}&nbsp;{{ file.lineCount }}&nbsp;({{ file.lineCount - file.blankLineCount }})
-        pre.hljs.file-content(v-if="file.wasCodeLoaded", v-show="file.active")
+          ) {{ file.fileType }}&nbsp;{{ file.lineCount }}&nbsp;({{ file.blankLineCount }})
```

Comment:
Lines like this look like old code before we made the changes

Reply:
Thanks for pointing it out. I have changed it accordingly.